build_tools/python/benchmark_suites/iree/cuda_benchmarks.py

```diff
@@ -31,12 +32,12 @@ def generate(
   ) -> Tuple[List[iree_definitions.ModuleGenerationConfig],
              List[iree_definitions.E2EModelRunConfig]]:
     """Generates IREE compile and run configs."""
-
+    models = model_groups.LARGE + [torch_models.MODEL_UNET_2D_FP32_TORCH]
```

Comment:
Should we define this as a group instead? @pzread

Reply:
Oh I see in the RISC-V benchmarks for instance, we define a constant with the relevant models. Let's do that here too (just slightly more obvious than having it inline IMO):

https://github.com/openxla/iree/blob/main/build_tools/python/benchmark_suites/iree/riscv_benchmarks.py#L17

Reply:
In practice, the model groups are tied heavily to the backend and what is supported in the frontend dialects so it might make sense to group based on backend.